.github/workflows/deploy.yml

```
@@ -31,6 +31,15 @@ jobs:
         uses: gittools/actions/gitversion/execute@v3
         with:
           useConfigFile: true
+
+      - name: Generate dist
+        run: |
+          cp -r app poc-php-mvc
+          cp -r public poc-php-mvc
+          cp -r src poc-php-mvc
+          mv .htaccess-prod poc-php-mvc/.htaccess
+          mv composer.json poc-php-mvc/composer.json
+          mv composer.lock poc-php-mvc/composer.lock          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)